src/components/footer/www/footer.jsx

```diff
@@ -4,165 +4,206 @@ var FormattedMessage = require('react-intl').FormattedMessage;
-                                <FormattedMessage id='general.scratchConference' />
-                            </a>
-                        </dd>
-                        <dd>
-                            <a href="http://www.scratchfoundation.org/">
-                                <FormattedMessage id='general.scratchFoundation' />
-                            </a>
-                        </dd>
-                    </dl>
-                </div>
+                        <dl>
+                            <dt>
+                                <FormattedMessage id='general.legal'/>
+                            </dt>
+                            <dd>
+                                <a href="/terms_of_use/">
+                                    <FormattedMessage id='general.termsOfUse' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="/privacy_policy/">
+                                    <FormattedMessage id='general.privacyPolicy' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="/DMCA/">
+                                    <FormattedMessage id='general.dmca' />
+                                </a>
+                            </dd>
+                        </dl>
 
+                        <dl>
+                            <dt>
+                                <FormattedMessage id='footer.scratchFamily' />
+                            </dt>
+                            <dd>
+                                <a href="http://scratched.gse.harvard.edu/">
+                                    <FormattedMessage id='general.scratchEd' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="http://www.scratchjr.org/">
+                                    <FormattedMessage id='general.scratchJr' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="http://day.scratch.mit.edu/">
+                                    <FormattedMessage id='general.scratchday' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="/conference/">
+                                    <FormattedMessage id='general.scratchConference' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="http://www.scratchfoundation.org/">
+                                    <FormattedMessage id='general.scratchFoundation' />
+                                </a>
+                            </dd>
+                        </dl>
+                    </div>
+                </MediaQuery>
```

Comment:
There's a lot of duplication here, which will make this annoying to maintain — especially if this is the example the rest of the site would need to follow.